docs/pages/router/advanced/anchor-routes.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Anchor Routes
+description: Learn how to use the anchor routes to modify navigation history
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+Anchor routes, sometimes referred to as initial routes, are routes that are injected into the start of the navigation history when navigator is loaded.
+
+Expo Router uses a different term to React Navigation due to the different philosophy of the API design. 
+
+- Expo Router uses a **positive** API: A route is assigned to be the anchor and it will be included by setting `withAnchor` to a positive value. 
+- React Navigation uses a **negative** API: A target route by default is the initial route unless `initial: false` is passed, in which case the default initial route is used.
+
+<FileTree
+  files={[
+    ['_layout.tsx'],
+    ['(feed)/_layout.tsx', 'A <Stack> layout'],
+    ['(feed)/profile/[id].tsx', 'A users profile'],
+    ['(feed)/index.tsx', 'A news feed'],
+  ]}
+/>
+
+In the example above, the `(feed)/index.tsx` route is assigned the `/` path and will be the first route shown. When the users navigates backwards the app will close as there is no history. The same logic also applies to deep links to `/profile/<id>`. This is generally undesirable behavior for mobile apps, as going back from a deep-link generally should route to the `/` route before existing. We can achieve this via assigning an anchor route:
```

Comment:
```suggestion
In the example above, the `(feed)/index` route is assigned the `/` path and will be the first route shown. When the user navigates backwards, the app will close as there is no history. The same logic also applies to deep links to `/profile/<id>`. This is generally undesirable behavior for mobile apps, as going back from a deep link generally should route to the `/` route before exiting. We can achieve this by assigning an anchor route:
```

- I think we can omit file format in the first line since we are directly referring to the route.
- Grammatical typo in the second line.